component/Page/Profile/index.tsx

```diff
@@ -35,6 +37,7 @@ export default function ProfilePage(props: PageProps) {
 
     useEffect(() => {
         if (isDynamicLink) router.push(`/${username}`, undefined, { shallow: true })
+        sendAnalytics(BetterSocialEventTracking.PROFILE_SCREEN_OPEN)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useEffect` hook is missing a dependency for `sendAnalytics`. This could lead to unexpected behavior if the function changes. Add it to the dependency array.
```diff
-    }, [isDynamicLink])
+    }, [isDynamicLink, sendAnalytics])
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->